dnddiceroller/DiceFileReader.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dnddiceroller;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+/**
+ *
+ * @author Paul Scarrone
+ */
+public class DiceFileReader {
+  private String filePath;
+  private Scanner input = null;
+  private List<String> lines;
+
+  /*
+    Use this with
+	DiceFileReader reader = new DiceFileReader("dice.txt")
+  */
+  
+  public DiceFileReader(String filePath) {
+	this.filePath = filePath;
+	
+	File inputFile = new File(filePath);
+	
+	try {
+	  this.input = new Scanner(inputFile);
+	} catch (FileNotFoundException ex) {
+	  Logger.getLogger(DiceFileReader.class.getName()).log(Level.SEVERE, null, ex);
+	}
+  }
+  
+  public List<String> getLines(){
+	return this.lines;
+  }
+  
+  public void read(){
```

Comment:
IMO roll should be the responsibility of DiceTower... but the only way that I can think of to make that work with the way I have it set up is to pass an arrayList of arrayLists to DiceTower and iterate through it in a method of DiceTower. I'm not sure if that's a real thing, and I couldn't get that to work so far.